pyproject.toml

```diff
@@ -36,3 +45,24 @@ addopts = "--ignore=test/integration_tests/resources"
 testpaths = [
     "test"
 ]
+
+[[tool.mypy.overrides]]
+module = ["exasol_integration_test_docker_environment.*", "luigi.*", "docker.*",
+            "humanfriendly", "configobj", "exasol.bucketfs", "toml"]
```

Comment:
Please review this list. I believe this project does not use luigi.
I recommend to start with an empty list and then add only those project which cause mypy failures.